address_book_query.go

```diff
@@ -191,3 +192,12 @@ func (query *AddressBookQuery) Execute(client *Client) (NodeAddressBook, error)
 		NodeAddresses: result,
 	}, subClientError
 }
+
+func (query *AddressBookQuery) SetLogLevel(level LogLevel) *AddressBookQuery {
+	query.logLevel = &level
+	return query
+}
+
+func (query *AddressBookQuery) GetLogLevel() *LogLevel {
+	return query.logLevel
+}
```

Comment:
This is not used anywhere because of the custom flow of this query. We should probably remove it for now

Reply:
Sure, for now it is not needed.

Reply:
removed